test/no-for-loop.js

```diff
@@ -482,6 +482,46 @@ ruleTester.run('no-for-loop', rule, {
 				const [ a, b ] = element;
 				console.log(a, b, element);
 			}
+		`),
+		// Singularization (simple case):
+		testCase(outdent`
+			for (let i = 0; i < plugins.length; i++) {
+				console.log(plugins[i]);
+			}
+		`, outdent`
+			for (const plugin of plugins) {
+				console.log(plugin);
+			}
+		`),
+		// Singularization (irregular case):
+		testCase(outdent`
+			for (let i = 0; i < people.length; i++) {
+				console.log(people[i]);
+			}
+		`, outdent`
+			for (const person of people) {
+				console.log(person);
+			}
+		`),
+		// Singularization (avoid using reserved JavaScript keywords):
+		testCase(outdent`
+			for (let i = 0; i < cases.length; i++) {
+				console.log(cases[i]);
+			}
+		`, outdent`
+			for (const element of cases) {
+				console.log(element);
+			}
+		`),
+		// Singularization (capital letters, multiple words):
+		testCase(outdent`
+			for (let i = 0; i < LARGE_CITIES.length; i++) {
+				console.log(LARGE_CITIES[i]);
+			}
+		`, outdent`
+			for (const LARGE_CITY of LARGE_CITIES) {
+				console.log(LARGE_CITY);
+			}
```

Comment:
Does the singularization also work for camel-case identifiers, e.g. `largeCities` ?

Reply:
Yep!

```
Welcome to Node.js v12.16.2.
> const pluralize = require('pluralize')
undefined
> pluralize.singular('largeCities')
'largeCity'
```

Reply:
That's great ! It would be nice to add an example and a test for this case.

Reply:
Added a camelCase test case.